content/blog/2021-05-24-radiobeitrag-deutschlandfunknova/index.de.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Der OpenBikeSensor auf Deutschlandfunk
+linkTitle: Der OpenBikeSensor auf Deutschlandfunk
+date: 2021-05-25
+---
+
+Bereits am 09. Mai 2021 gab es einen Beitrag über den OpenBikeSensor in der Sendung [Netzbasteln](https://www.deutschlandfunknova.de/serien/netzbasteln) auf Detschlandfunk Nova.
+Der Deutschlandfunk-Nova-Reporter Moritz Metz hat selbst einen OpenBikeSensor zusammengebaut und diesen "live" auf einer Bundesstraße in Brandenburg getestet.
+
+Der ganze Beitrag findet sich auf [deutschlandfunknova.de](https://www.deutschlandfunknova.de/beitrag/radsicherheit-abstandsmesser-selber-bauen).
+Auch [auf Twitter](https://twitter.com/netzbasteln) hat Moritz Metz fleißig über den OpenBikeSensor berichtet.
```

Comment:
Can the link to twitter be more specific than just the user? Maybe a search to all tweets with `#openbikesensor` by that user? Or does twitter not have that search as a feature?

Reply:
Ich finde gerade keine Option...